### PR TITLE
[zh] sync page cluster-level-pss

### PR DESCRIPTION
**Pull request by Zhuzhenghao**
This PR resyncs `content/zh-cn/docs/tutorials/security/cluster-level-pss.md`.

### CODE REVIEW

File: content/zh-cn/docs/tutorials/security/cluster-level-pss.md [hunk 1]
```diff
@@ -57,6 +57,16 @@ Install the following on your workstation:
 - [KinD](https://kind.sigs.k8s.io/docs/user/quick-start/#installation)
 - [kubectl](/zh-cn/docs/tasks/tools/)
 
+<!--
+This tutorial demonstrates what you can configure for a Kubernetes cluster that you fully
+control. If you are learning how to configure Pod Security Admission for a managed cluster
+where you are not able to configure the control plane, read
+[Apply Pod Security Standards at the namespace level](/docs/tutorials/security/ns-level-pss).
+-->
+本教程演示了你可以为你完全控制的 Kubernetes 集群配置的内容。
```
Comment (tengqm) [inline content/zh-cn/docs/tutorials/security/cluster-level-pss.md:66]:
```suggestion
本教程演示了你可以对完全由你控制的 Kubernetes 集群所配置的内容。
```

File: content/zh-cn/docs/tutorials/security/cluster-level-pss.md [hunk 2]
```diff
@@ -57,6 +57,16 @@ Install the following on your workstation:
...
+如果你正在学习如何为一个无法配置控制平面的托管集群配置 Pod 安全准入，
+请参阅[在名字空间级别应用 Pod 安全标准](/zh/docs/tutorials/security/ns-level-pss)。
```
Comment (tengqm) [inline content/zh-cn/docs/tutorials/security/cluster-level-pss.md:68]:
```suggestion
请参阅[在名字空间级别应用 Pod 安全标准](/zh-cn/docs/tutorials/security/ns-level-pss)。
```

File: content/zh-cn/docs/tutorials/security/cluster-level-pss.md [hunk 3]
```diff
@@ -164,7 +184,10 @@ that are most appropriate for your configuration, do the following:
       kubectl label --dry-run=server --overwrite ns --all \
       pod-security.kubernetes.io/enforce=baseline
       ```
-      <!-- The output is similar to this: -->
+
+      <!--
+      The output is similar to:
+      :-->
```
Comment (windsonsea) [inline content/zh-cn/docs/tutorials/security/cluster-level-pss.md:190]:
```suggestion
      -->
```
This line has problem.

image

File: content/zh-cn/docs/tutorials/security/cluster-level-pss.md [hunk 4]
```diff
@@ -183,7 +206,10 @@ that are most appropriate for your configuration, do the following:
       kubectl label --dry-run=server --overwrite ns --all \
       pod-security.kubernetes.io/enforce=restricted
       ```
-      <!-- The output is similar to this: -->
+
+      <!--
+      The output is similar to:
+      -->
       输出类似于：
```
Comment (windsonsea) [inline content/zh-cn/docs/tutorials/security/cluster-level-pss.md:213]:
```suggestion
      <!--
      The output is similar to:
      -->
      
      输出类似于：
      
```
This line has problem:

image

**Comment by windsonsea** [comment]
The [preview](https://deploy-preview-39803--kubernetes-io-main-staging.netlify.app/zh-cn/docs/tutorials/security/cluster-level-pss/) looks good now, thanks.
/lgtm

**Pull request merged**